isis/src/tgo/apps/tgocassis2isis/tgocassis2isis.cpp

```diff
@@ -81,12 +90,33 @@ void translateCoreInfo(FileName &inputLabel, ProcessImport &importer) {
   PvlGroup &dataDir = Preference::Preferences().findGroup("DataDirectory");
   QString missionDir = (QString) dataDir["Tgo"];
 
+  // How to tell that this is an ISIS3 exported product (generated by tgocassisrdrgen?)
```

Comment:
i have added something to ProcessExportPds4 to address this issue (for my hyb1 ticket). You can remove this comment.